platform-sdk/docs/proposals/TSS-Ledger-Id/TSS-Ledger-Id.md

```diff
@@ -0,0 +1,346 @@
+# Threshold Signature Scheme (TSS) Ledger ID
+
+---
+
+## Summary
+
+This proposal is for the integration of a threshold signature scheme into the consensus nodes to allow them to create a
+private/public key pair for the ledger that can be used to sign blocks and construct block proofs. The private key must
+be a secret that no one knows. The public key is known by all and functions as the ledger id. The consensus nodes must
+be able to aggregate their individual signatures on a message into a valid ledger signature for the message that is
+verifiable by the public key.
+
+| Metadata           | Entities                                   | 
+|--------------------|--------------------------------------------|
+| Designers          | Edward, Cody, Austin, Kore, Anthony, Artem |
+| Functional Impacts | Services, DevOps                           |
+| Related Proposals  | TSS-Library, TSS-Roster, TSS-Block-Signing |
+| HIPS               | N/A                                        |
+
+---
+
+## Purpose and Context
+
+Users of a ledger want to have cryptographic proofs that a statement was true on the ledger at a particular point in
+time. Our chosen method for providing these types of proofs is to assign a permanent public/private key pair to the
+ledger for use in signing blocks and constructing block proofs. The ledger private key must be a secret that no one
+knows. The ledger public key is known by all and functions as the ledger id. While the ledger private key is unknown to
+anyone, the consensus nodes must be able to aggregate their individual signatures on a message into a valid ledger
+signature for the message that is verifiable by the public key.
+
+The TSS effort has been broken down into four separate proposals: TSS-Library, TSS-Roster, TSS-Ledger-Id, and
+TSS-Block-Signing.
+
+1. The `TSS-Library` proposal contains the cryptographic primitives and algorithms needed to implement TSS.
+2. The `TSS-Roster` proposal introduces the data structure of a consensus `Roster` to replace the platform's concept of
```

Comment:
One point of this proposal that I do not understand is the consensus roster. Would it be possible to add a short paragraph describing what that is?

Reply:
I will leave that for the `TSS-Roster` proposal to be more descriptive since this proposal is dependent on it.